src/crimson/os/seastore/onode_manager/staged-fltree/node.cc

```diff
@@ -191,6 +191,8 @@ node_future<Ref<Node>> Node::load_root(context_t c, RootNodeTracker& root_tracke
                  &root_tracker](auto root) mutable {
       assert(root->impl->field_type() == field_type_t::N0);
       root->as_root(std::move(_super));
+      std::ignore = c; // as only used in an assert
```

Comment:
How about [`[[maybe_unused]]`](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused)? It can be applied for lambda's parameters.

Reply:
Forgot that. Will fix. Thanks

Reply:
@rzarzynski: Well - it's actually not allowed.
So I suggest using the std::ignore.

Reply:
Oops, it really looks `[[maybe_unused]]` won't work with closures. My fault, let's merge as is.